pawn/Elements/AdministratorCommands.pwn

```diff
@@ -1017,7 +1017,12 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "The pirate ship is a peace zone! Please don't hit, shoot or throw grenades on it!");
     }
-
+    
+    else if (!strcmp(showParameter, "spraytag", true, 8)) {
+        showInfo = true;
+        format(g_message, sizeof(g_message), "Want to Spawn vehicles like /tur, /inf, /vor, /nrg , /sul ? . Complete Spraying all the 100 Spraytags found across LV and have fun spawning vehicles!");
```

Comment:
This message is too long, and contains a series of typos. How about:

"Spray all 100 tags found around Las Venturas, and get the ability to spawn your own vehicles!"